### DISCUSSION TITLE
what is /runnertmp for

### DISCUSSION THREAD

**Discussion by shichengripple001** [actions/actions-runner-controller#1110] [Questions] [answered]
I am trying to build some cutom image, I want to understand how is /runnertmp being copied into /runner folder. i don't see anything about /runner folder in Dockerfile, but services are seemed to be run from /runner folder.

**Comment by toast-gear** [comment, marked as answer]
https://github.com/actions-runner-controller/actions-runner-controller/blob/master/runner/entrypoint.sh#L83

**Reply by shichengripple001** [reply]
thank you for the info!
